kafka-bom/build.gradle.kts

```
@@ -4,8 +4,8 @@ plugins {
 }
 
 
-var kafkaVersion = "7.5.2"
-var kafkaCcsVersion = "$kafkaVersion-ccs"
+var confluentVersion = "7.7.0"
```

```suggestion
var confluentKafkaVersion = "7.7.0"
```
Just in case if later any other confluent dependencies get introduced.

---

It's confluent platform version. It should be same for all the confluent dependencies.